skatepark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "skatepark",
-  "version": "1.0.0",
+  "version": "1.1.1",
```

Comment:
Judging from #4492, I actually don't think we should bump this.

Reply:
Indeed!  Looks like the incorrect version just came along in the initial commit for Skatepark's style.css file.